src/main/java/com/rpietraszewski/medicalclinic/model/entity/Doctor.java

```diff
@@ -0,0 +1,47 @@
+package com.rpietraszewski.medicalclinic.model.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "DOCTOR")
+@Entity
+public class Doctor {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "DOCTOR_ID")
+    private Long doctorId;
+    @Column(name = "EMAIL", unique = true)
+    private String email;
+    @Column(name = "PASSWORD")
+    private String password;
+    @Column(name = "FIRST_NAME")
+    private String firstName;
+    @Column(name = "LAST_NAME")
+    private String lastName;
+    @Column(name = "SPECIALIZATION")
+    private String specialization;
+    @ManyToMany
+    @JoinTable(
+            name = "DOCTOR_INSTITUTION",
+            joinColumns = @JoinColumn(name = "DOCTOR_ID"),
+            inverseJoinColumns = @JoinColumn(name = "INSTITUTION_ID"))
+    private Set<Institution> institutions;
```

Comment:
private Set<Doctor> doctors = new HashSet<>();
<- dzieki temu jak hibernate bedzie ci przemapowywal wiersze z bazy danych na encje to jesli nie znajdzie encji to i tak zwroci ci pusty set a nie nulla.